public/index.js

```diff
@@ -256,13 +256,26 @@
     function addArrayToStorage(key, array){
       sessionStorage.setItem(key, JSON.stringify(array));
     }
+
+    function updateSessionStorage (key, node, array, className) {
+      if (!node) return;
+      node.addEventListener('click', function () {
+        if ( node.className.baseVal === className ) {
+          array.push(node.id);
```

Comment:
Okay I do not know what is going on here, but this mutation is not good, given that this function is meant to put something in local storage, you would not expect it to also mutate one of its input arguments.

Have we got two sources of truth? Ie local variable, personality, which gets mutated, and also what in local storage, what is happening here generally seems bad, but I am not sure I have time to go back and understand this file..

Does what I am saying make sense?